src/libraries/Microsoft.Extensions.Configuration.Binder/README.md

```diff
@@ -89,3 +89,42 @@ You can include a configuration file using a code like this in your `.csproj` fi
   </Content>
 </ItemGroup>
 ```
+
+## Contributing changes to the source generator
+
+Along with the binder assembly that uses reflection, we ship a source generator assembly that provides a reflection and AOT compatible implementation. It works by intercepting regular binding calls and redirecting them to new routines that execute strongly typed binding logic.
+
+### Worfklow
+
+- [Build the .NET libraries and runtime assemblies](https://github.com/dotnet/runtime/tree/7ed33d80c92fa0b7ae740df60a460e984f2f442b#how-can-i-contribute)
```

Comment:
I was trying to see if we could skip this step.  While the generator itself can be built directly without building the entire repo, the tests cannot -- since they run on the live-built framework.

Reply:
👍 

We wouldn't need this step for regression tests between major releases. I imagine a frozen sample app using the new TFM, so we'd build just the app & the live-built generator. Is this what you had in mind in our earlier convo? I gather that's what JSON does but I can't seem to find those tests. cc @eiriktsarpalis

Reply:
Should be this one: 

https://github.com/dotnet/runtime/tree/main/src/libraries/System.Text.Json/tests/System.Text.Json.Tests/SourceGenRegressionTests